runner/src/run.rs

```diff
@@ -46,6 +57,11 @@ pub fn run(args: &RunArgs) -> ExitCode {
         None => return ExitCode::FAILURE,
     };
 
+    // In case we run Ubuntu, we need to download the ubuntu image
+    if args.firmware == "ubuntu" {
+        fetch_ubuntu_image();
+    }
```

Comment:
We don't want to hardcode this, if we need to add more disk or test with different firmware this is not going to scale well.